README.md

```diff
@@ -1 +1,110 @@
-# ansible-role-k8s-web-cluster
+# caktus.k8s-web-cluster
+
+An Ansible role to help set up Kubernetes clusters for web apps.
+
+
+# License
+
+This Ansible role is released under the BSD License. See the
+[LICENSE](https://github.com/caktus/ansible-role-k8s-web-cluster/blob/master/LICENSE)
+file for more details.
+
+Development sponsored by [Caktus Consulting Group, LLC](http://www.caktusgroup.com/services>).
+
+
+# Requirements
+
+* ``pip install openshift kubernetes-validate``
+
+
+# Installation
+
+1. Add to your ``requirements.yml``:
+
+
+```yaml
+---
+# file: deployment/requirements.yaml
+
+- src: https://github.com/caktus/ansible-role-k8s-web-cluster
+  version: init-role  # TODO: remove
+  name: caktus.k8s-web-cluster
+```
+
+2. Add the role to your playbook:
+
+```yaml
+---
+# file: playbooks/configure-cluster.yaml
+
+- hosts: k8s_clusters
+  roles:
+    - role: caktus.k8s-web-cluster
+```
+
+3. Add role vars configuration:
+
+```yaml
+
+---
+# ----------------------------------------------------------------------------
+# caktus.k8s-web-cluster: Configure kubernetes cluster for web apps
+# ----------------------------------------------------------------------------
+
+k8s_cluster_type: <aws|gcp|azure|digitalocean>
+k8s_context: <name of context from ~/.kube/config>
+k8s_letsencrypt_email: <email to contact about expiring certs>
+
+k8s_echotest_hostname: <test hostname assigned to your cluster ip, e.g. echotest.caktus-built.com>
+```
+
+4. Run ``playbooks/configure-cluster.yaml``
+
+
+### Testing that Let's Encrypt is working
+
+1. Find the hostname or IP of your load balancer.
+2. Add a DNS record for ``k8s_echotest_hostname`` to
+   point to this hostname or IP address (switching the record type if needed).
+3. Give the record a minute or two to propagate.
+4. Add an echotest playbook:
+
+```yaml
+---
+# file: playbooks/echotest.yaml
+
+- hosts: k8s_clusters
+  tasks:
+    - name: Install echo test server
+      import_role:
+        name: caktus.k8s-web-cluster
+        tasks_from: echotest
+```
+
+5. Run ``playbooks/echotest.yaml``
```

Comment:
Would it be better to give the whole command here? Or by this point can we assume most of us are generally familiar with how to run an Ansible playbook?